### DISCUSSION TITLE
選舉局勢

### DISCUSSION THREAD

**Discussion by Ryanlolry** [1101-GS4719-C/Python-Community-GS4719-C#1747] [General]
在藍白合破局及郭台銘退選後，賴清德已經穩上了，除非有什麼大變動，向陳水扁被槍擊那樣足以影響一切的事件。
但我認為應該給聯合政府一個機會，畢竟藍綠兩黨都用實力證明，垃圾不分藍綠，要打倒自己人，只要有利益，有無反顧，只要對他們有害，就毫不理會。
所以如果能給聯合政府一個機會，讓彼此的利益互相碰撞，不讓一個政黨獨大，會不會是一個比較好的場面，會不會對台灣比較好。

**Comment by Chengbosco** [comment]
我也認為現在的政治局勢太過誇張了，所以我認為國會絕對不能讓任何一黨超過半數，不然會變成該過的法都不過，有利益的法強制輾壓，希望這次至少能平均分散，讓他們利益限縮、碰撞。

**Comment by Ryanlolry** [comment]
的確，他們用實力證明，不能再輕信他們的鬼話，他們都完全執政過，但都超爛，所以改變是勢在必行，所以可以試試看讓立法院成為三足鼎立的情勢，但也會有問題，可能會形成更他的衝突，希望他們可以好好解決，希望有更好的台灣。

韓國瑜：台灣發大財
